websites/views.py

```diff
@@ -315,6 +315,23 @@ def pipeline_status(self, request, name=None):
         )
         return Response(status=200)
 
+    @action(detail=True, methods=["get"], permission_classes=[BearerTokenPermission])
+    def allow_offline_build(self, request, name=None):  # noqa: ARG002
+        """Process webhook requests from concourse pipeline runs"""
+        website = get_object_or_404(Website, name=name)
+        content = WebsiteContent.objects.get(
+            website=website, type=CONTENT_TYPE_METADATA
+        )
+        allow_offline = content.metadata["hide_download"]
```

Comment:
Also, why not just call this variable `hide_download`? It would more accurately reflect what it represents. Realistically, a variable called `allow_offline` would actually be the opposite of `hide_download`.